### PR TITLE
Update organizers

### PR DESCRIPTION
**Pull request by louis993546**
[Click here for demo](https://devfest-berlin-2019-testing-a.firebaseapp.com/team/)

-----------------

Technically still in progress, but would like to open for review + encourage external contributions 🙏

### CODE REVIEW

File: docs/default-firebase-data.json [hunk 1]
```diff
@@ -394,67 +491,392 @@
...
+              "name": "Twitter"
+            }
+          ],
+          "order": 2,
+          "title": ""
+        },
+        {
+          "name": "Eden",
+          "photo": "",
+          "photoUrl": "https://devfest-berlin-2019-testing-a.firebaseapp.com/images/team/",
```
Comment (mariobodemann) [inline docs/default-firebase-data.json:545]: That is a wrong URL. If we don't want it, maybe we can skip it? 🙈

File: docs/default-firebase-data.json [hunk 2]
```diff
@@ -728,14 +1179,17 @@
...
-          "tags": [],
-          "title": "Android Things 🛠🤖🛠",
-          "videoId": ""
-        }]
+        "2017": [
+          {
+            "presentation": "https://docs.google.com/presentation/d/1ByepX6zNVZOTJnMEl01rYhaKvN-TzoeO-B-IwhAsoUs",
+            "tags": [],
+            "title": "Android Things 🛠🤖🛠",
+            "videoId": ""
```
Comment (mariobodemann) [inline docs/default-firebase-data.json:1197]: Not sure if we still need this. But maybe it's just a place holder?

File: docs/default-firebase-data.json (hunk 1 above)
Comment (louis993546) [reply docs/default-firebase-data.json:545]: yea we can skip it for sure 👍

File: docs/default-firebase-data.json (hunk 2 above)
Comment (louis993546) [reply docs/default-firebase-data.json:1197]: Probably just a placeholder, let me check......

**Comment by louis993546** [comment]
This one is no longer necessary, see [this repo](https://github.com/devfest-berlin/2019.devfest-berlin.de/) instead.

**Pull request closed without merging**